.eslintrc.json

```diff
@@ -19,7 +19,9 @@
       "react/react-in-jsx-scope": "off",
       "import/no-unresolved": "off",
       "no-shadow": "off",
-      "no-param-reassign": "off"
+      "no-param-reassign": "off",
+      "no-unused-vars": "off",
+      "react/jsx-props-no-spreading": "off"
```

Comment:
- [ ]  Great job so far, Kindly don't edit linters, if errors come up let's try and find a way to fix them rather than circumvent the situation.

Reply:
@Wes-Isaac If you can kindly assist me here. How do I handle the **_no-param-reassign_**? All the tutorials we were given use parameter assignment so, how do you suggest I handle it? Looking forward to your feedback. Thanks